cms/templates/cms/home_page.html

```diff
@@ -217,31 +165,80 @@ <h3>Why should I enroll?</h3>
                <img src="{% static 'images/online.png' %}" alt="">
             </div>
 
-            <h4>Convenient &amp; Flexible</h4>
-            <p>Take online courses on a schedule that fits your needs, and meet motivated learners who share your interests.</p>
+            <h4>Advance your career</h4>
+            <p>Learn online, pass digital proctored exams, and earn a program credential. 
+              Boost your professional profile with an affordable and valuable credential from MIT.</p>
           </div>
           <div class="enroll-reason col-md-3">
             <div class="reasons-images-wrap">
               <img src="{% static 'images/blended.png' %}" alt="">
             </div>
 
             <h4>Accelerate Your Master’s</h4>
-            <p>A MicroMasters program is a faster path to a master’s degree from MIT and other universities, and is recognized by employers for providing deep learning in specific career fields.</p>
+            <p>Fast-track your master’s degree at MIT and  many other <a href="/pathways-graduate-programs/">pathway schools</a> around the world.</p>
 
           </div>
           <div class="enroll-reason col-md-3">
             <div class="reasons-images-wrap">
-              <img src="{% static 'images/cost.png' %}" alt="">
+              <img src="{% static 'images/dome_icon.webp' %}" alt="">
             </div>
-             <h4>Save Money &amp; Time</h4>
-            <p>A MicroMasters program is an affordable option for professionals who can’t take off time to earn a traditional master’s degree.</p>
+             <h4>Be part of the MIT community</h4>
+            <p>Become an affiliate member of the MIT Alumni Association and receive special access to professional journals and publications.</p>
```

Comment:
```suggestion
            <p><a href="/benefits/">Become an affiliate member of the MIT Alumni Association</a> and receive special access to professional journals and publications.</p>
```

Move the link from the button up here, because the `/benefits/` page is really only about the Alumni Association.

Reply:
@umarmughal824 is this really resolved? as I am unable to see any change.

Reply:
Sorry, this was one of the cases where the team changed their minds back and forth. The current implementation, with a button that links to `/benefits/` and no linked text in the "Be Part of the Community" section is preferred.